src/lib/services/BidService.js

```diff
@@ -179,7 +183,7 @@ export default class BidService {
   }
 
   extendBid(endsAt, receivedAt) {
-    endsAt = endsAt || new Date()
+    endsAt = endsAt || daysFromNowToDate(env.get('MIN_GRACE_PERIOD_DAYS', 14))
```

Comment:
It doesn't seem to be working as `daysFromNowToDate` doesn't return a `Date` but a number representing a timestamp!